docs/contracts.rst

```diff
@@ -22,43 +22,66 @@ To run this example, you will need to install a few extra features:
 .. code-block:: python
 
     import json
-    import web3
-
+    
     from web3 import Web3
-    from solc import compile_source
+    from solc import compile_standard
+    from web3.contract import ConciseContract
 
     # Solidity source code
-    contract_source_code = """
-    pragma solidity ^0.4.21;
+    compiled_sol = compile_standard(
+    {
+        "language": "Solidity",
+        "sources": {
+            "Greeter.sol": {
+                "content": '''
+    pragma solidity ^0.5.0;
```

Comment:
Can we move this `content` so that it is indented 4 spaces under content please?